lib/metrics/common_fields.go

```diff
@@ -0,0 +1,31 @@
+package metrics
+
+import (
+	"database/sql"
+
+	"github.com/status-im/telemetry/pkg/types"
+)
+
+func InsertTelemetryRecord(tx *sql.Tx, data *types.TelemetryRecord) (int, error) {
+	stmt, err := tx.Prepare(`
+		INSERT INTO telemetryRecord (nodeName, peerId, statusVersion, deviceType)
+		VALUES ($1, $2, $3, $4)
+		RETURNING id;
+	`)
```

Comment:
As described in https://github.com/status-im/telemetry/issues/43 consider using exec, or if this insertion is going to be called frequently, maybe it's a good idea to extract the prepared statement and make it available during the entire duration of the program.